Apps/ValidationTests/macOS/ViewController.mm

```diff
@@ -75,7 +76,7 @@ - (void)refreshBabylon {
         Babylon::Polyfills::XMLHttpRequest::Initialize(env);
 
         graphics->AddToJavaScript(env);
-        Babylon::Plugins::NativeEngine::Initialize(env, false); // render on UI Thread
+        Babylon::Plugins::NativeEngine::Initialize(env, RENDER_ON_JS_THREAD);
```

Comment:
We are now changing this from false to true. What is the reason why it doesn't work?

Reply:
Because of this : https://github.com/BabylonJS/BabylonNative/issues/512

Reply:
👍